esp/src/src-react/components/ECLArchive.tsx

```diff
@@ -72,6 +79,10 @@ export const ECLArchive: React.FunctionComponent<ECLArchiveProps> = ({
         pushUrl(`${parentUrl}/${selId}`);
     }, [parentUrl]);
 
+    const onChangeTreeFilter = React.useCallback((event: React.FormEvent<HTMLInputElement | HTMLTextAreaElement>, newValue?: string) => {
+        setTreeFilter(newValue || "");
```

Comment:
`newValue ?? ""` the other instance might need fixing as well?

Reply:
Changed both places using this on the metrics page also. I went ahead and changed a handful of places in the Sasha admin component that was using this incorrectly as well.